cmd/validators/validators.go

```diff
@@ -15,6 +16,7 @@ import (
 // Validate is a validator for commands to print an error when the user input is invalid.
 func Validate() cobra.PositionalArgs {
 	return func(cmd *cobra.Command, args []string) error {
+		rebindFlags(cmd, cmd.ValidArgs) // rebind flags before validating them below
```

Comment:
`Validate` gets called before any of the callback methods on a command, so we have to rebind this early to make sure the flags below are set with what's in viper. Otherwise, cobra doesn't know that the flags were set from environment variables.